ui-tests/src/test/java/org/openmrs/reference/AddProviderTest.java

```diff
@@ -26,14 +26,14 @@
     private String personUuid;
 
     @Before
-    public void setUp() {
+	public void setUp() throws Exception {
```

Comment:
clean up the formatting issues, same comment here, any need to throwing the exception here?